src/wara_ps_client.h

```diff
@@ -50,16 +57,12 @@ class WaraPSClient {
     WaraPSClient(std::string name, std::string server_address);
 
     /**
-     * Deleted constructors to disallow copying and moving of the client
+     * Deleted constructors to disallow copying and assignment of the client
      */
-    WaraPSClient(WaraPSClient &&) = delete;
-
     WaraPSClient(const WaraPSClient &) = delete;
```

Comment:
Declaring this will also automatically never declare the move constructor.

Reply:
Very nice! Didn't know this.